### PR TITLE
Add silent mode

### LINKED ISSUE #13: No animation/Silent mode
If I run Pydoro in a visible iTerm pane, the timer animation is distracting. Perhaps having a "silent mode" would be nice, just animate when done, so as to break distraction less.

### PR DESCRIPTION
**Pull request by gmelodie**
The functionality is not ready yet. I've added some code to recognize the `--silent` flag in the command line.

**Problems:**
1. Should the silent mode run only one pomodoro and exit?
2. Should it run one pomodoro and then draw the application?
3. Should it have a different layout without the clock?

Fixes #13

### CODE REVIEW

**Comment by JaDogg** [comment]
* Should the silent mode run only one pomodoro and exit? - no
* Should it run one pomodoro and then draw the application? - no
* Should it have a different layout without the clock? - Same layout just no clock or progress bar. But some animation when the task is done is good. 

However this doesn't stop the sound from playing. So now I think `--no-animation` might also be a better argument as `silent` seems like no sound mode. 🤔 What do you think?

**Comment by gmelodie** [comment]
It wouldn't be hard to implement something more complete, like 

- `--no-animation`
- `--no-sound`
- `--focus` (both `no-animation` and `no-distraction`)

Another issue is how the menu is going to be displayed. I'm guessing it remains the same, just omitting the clock (right?), or omitting seconds and only displaying remaining minutes.

**Comment by JaDogg** [comment]
just omitting the clock (right?) - Yes.

**Comment by gmelodie** [comment]
All good on my end. `focus` and `no-clock` work like a charm, but I wasn't able to test `no-sound`. Can you test them for me?

**Comment by JaDogg** [comment]
Yes I'll test it and merge. 👍 Thank you.
Can you tell me what OS you are on? FYI if it is Linux or OSX you need to install `audio` requirements. see `setup.py` (For Windows this work out of the box)

File: pydoro/pydoro_tui.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env python
 import threading
 
+import argparse
```
Comment (supermitch) [inline pydoro/pydoro_tui.py]: isort this. ArgParse should be at the top of the file.

File: pydoro/pydoro_tui.py [hunk 2]
```diff
@@ -90,6 +92,25 @@ def draw():
...
+                        action="store_true")
+    args = parser.parse_args()
+
+    # Check for no-clock (or focus mode)
+    if args.no_clock or args.focus:
+        tomato._no_clock = True
+
+    # Check for no-sound (or focus mode)
+    if args.no_sound or args.focus:
+        tomato._no_sound = True
```
Comment (supermitch) [inline pydoro/pydoro_tui.py]: It's kinda weird to set a private attribute, no? Isn't that why we created it? Can you initialize the tomato object with it's config?

Comment (JaDogg) [reply pydoro/pydoro_tui.py]: I did change it to non private actually :)

File: pydoro/pydoro_tui.py (hunk 1 above)
Comment (JaDogg) [reply pydoro/pydoro_tui.py]: Indeed. Seems like PyCharm did it for me 👍

**Pull request merged**